shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/controllers/ManagedBridgeController.java

```diff
@@ -57,9 +53,6 @@ public class ManagedBridgeController implements Reconciler<ManagedBridge>,
     @ConfigProperty(name = "event-bridge.managed-bridge.poll.interval-milliseconds")
     int reconcileIntervalMillis;
 
-    @ConfigProperty(name = "event-bridge.managed-bridge.deployment.timeout-seconds")
```

Comment:
@vaibhavjainwiz Do you intend to re-add this at some point? I'm fine for it to be in another PR, but we just need to make sure we don't forget to do it

Reply:
timeout logic would moved to the control plane. It would not required in data-plane from now. 
I had created below jira for it
https://issues.redhat.com/browse/MGDOBR-1332